frontend/src/v5/ui/controls/progressBar/progressBar.component.tsx

```diff
@@ -0,0 +1,42 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { LinearProgress } from '@material-ui/core';
+import React from 'react';
+import { LabelledProgressBar, ProgressBarLabelLight, ProgressBarLabelDark } from './progressBar.styles';
+
+type IProgressBar = {
+	progress: number;
+	uploadStatus: 'queued' | 'uploading' | 'uploaded' | 'failed';
```

Comment:
Lets change the nae of the control to uploadProgressBar because it works using uploadStatus semantics.